core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -383,66 +386,68 @@ class ReplicaManager(val config: KafkaConfig,
                                messagesPerPartition: Map[TopicAndPartition, MessageSet],
                                requiredAcks: Short): Map[TopicAndPartition, LogAppendResult] = {
     trace("Append [%s] to local log ".format(messagesPerPartition))
-    messagesPerPartition.map { case (topicAndPartition, messages) =>
-      BrokerTopicStats.getBrokerTopicStats(topicAndPartition.topic).totalProduceRequestRate.mark()
-      BrokerTopicStats.getBrokerAllTopicsStats().totalProduceRequestRate.mark()
-
-      // reject appending to internal topics if it is not allowed
-      if (Topic.InternalTopics.contains(topicAndPartition.topic) && !internalTopicsAllowed) {
+    messagesPerPartition.map { case (topicAndPartition, messages) => {
+      {
```

Comment:
I'm a bit confused about the several braces that we now have after `map` (3 of them instead of the 1 we had before).